### PR TITLE
[PREVIEW] Lock nodejs to version 8.x

### PR DESCRIPTION
**Pull request by dwaynebailey**
Rationale:
- WebApps default available is 8.9.4 but 8.11.1 is available, so keep up
  with what is available on Azure
- Docker based images are always the latest on the 8.x branch, with
  nothing earlier.

https://tools.hmcts.net/jira/browse/RDM-3073

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +96,8 @@
     "sinon-chai": "^2.13.0",
     "sonar-scanner": "^3.0.3",
     "supertest": "^3.0.0",
-    "tslint": "^5.7.0"
+    "tslint": "^5.7.0",
+    "webpack": "^2"
```
Comment (vlaurin) [inline package.json]: Why is that needed?

Comment (dwaynebailey) [reply package.json]:
```
Your branch is up to date with 'origin/master'.
~/dev/moj/repos/ccd-case-print-service (master)$ yarn install
yarn install v1.9.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning " > karma-webpack@2.0.2" has unmet peer dependency "webpack@^1.1.0 || ^2 || ^2.1.0-beta.0 || ^2.2.0-rc.0".
warning "karma-webpack > webpack-dev-middleware@1.12.2" has unmet peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
```
